halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/BakeryController.java

```diff
@@ -195,6 +195,7 @@
     UpdateRequestBuilder builder = new UpdateRequestBuilder();
     builder.setSeverity(severity);
 
+    // TODO(lwander): Would be good to indicate when an added base image id conflicts with an existing base image id.
```

Comment:
I have mixed feelings about it. It will make the halyard approach simpler, but rosco will be unusable unless base images are first defined. I guess that's not really any different than the other services. So, sounds good.